opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/EventRestService.java

```diff
@@ -258,6 +263,14 @@ private void processEventAck(final SecurityContext securityContext, final OnmsEv
         m_eventDao.save(event);
     }
 
+    @POST
+    @Consumes({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON, MediaType.APPLICATION_ATOM_XML})
+    @Transactional
+    public Response publishEvent(final org.opennms.netmgt.xml.event.Event event) {
+        m_eventForwarder.sendNow(event);
```

Comment:
That's not the way I would do it, but guess it should work. It is up to you. Check how I implemented a similar thing on the ReST API for NBIs.